config.cfg

```diff
@@ -0,0 +1,23 @@
+[prod]
+port = 8000
+host = 0.0.0.0
+development = False
+remote_path = rereco_submission/
+service_url = https://cms-pdmv.cern.ch/grasp
+cmsweb_url = https://cmsweb.cern.ch
+credentials_file = ...
+database_auth = ...
+grid_user_cert = ...
+grid_user_key = ...

```

Comment:
Hm, I don't think we use most of these values, like `remote_path`, `service_url`, `cmsweb_url`, `credentials_file`, `database_auth` (maybe this will be relevant in the future), `grid_user_...`. Most of these are used for either pulling data from ReqMgr2 or DBS or SSH'ing into other machines. `port`, `host`  and `development` can stay, others are unused.